src/styles/dash_body.css

```diff
@@ -0,0 +1,36 @@
+.dash-main {
+  background-color: #101214;
+  color: #eee;
+  min-height: 100vh;
+  min-width: 100vh;
+  margin: 0px;
+  padding: 0;
+  position: relative;
+}
+
+.dash-main h1 {
```

Comment:
Please check to see if we have SASS available to us in `create-react-app`.  If so, update to:
```css
.dash-main {
  /* dash-main stuff */

  h1 {
    padding: 10px;
    /* etc. */
  }
}
```